src/pages/Book.jsx

```diff
@@ -1,15 +1,92 @@
-import { Link, useLocation } from "react-router-dom";
+import { Link, useLocation, useNavigate, useParams } from "react-router-dom";
+import axios from "axios";
 import "../styles/Book.css";
+import { useEffect, useState } from "react";
 
 function Book() {
+  const { id } = useParams();
   const location = useLocation();
   const { book } = location.state;
+  const [chapters, setChapters] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const getListOfChapters = async () => {
+      try {
+        const res = await axios.post(
+          "https://api-preprod.lelivrescolaire.fr/graph",
```

Comment:
This is used on several screens, so it could be a `const`. That way, if it changes later on, you won't have to look for it everywhere in the app to update it.